src/punctuation2.js

```diff
@@ -355,6 +368,8 @@ const addPairedPunctuationRules = {
   '710': addX10,
   '711': addX11,
   '730': addX30,
+  '773': addLinkingEntry,
```

Comment:
We have remover also for 774 and 776? Should we have adder too?

Reply:
Should. Punctuation code is patchy, and extended as time goes by. The total number of different combinations is next to infinite, so I've been adding things when the need arises, and not tried to cover it all.